client/src/charts/wrapped_nivo/WrappedNivoCircleProportion/WrappedNivoCircleProportion.js

```diff
@@ -119,101 +47,84 @@ export class WrappedNivoCircleProportion extends React.Component {
-      // so we need to remove the inner portion from the total here
-      value: parent_value - child_value,
-      color: color_scale(parent_name),
-      children: child_value
-        ? [
-            {
-              id: child_name,
-              name: child_name,
-              value: child_value,
-              color: color_scale(child_name),
-            },
-          ]
-        : 0,
+    const Circles = () => {
+      // arbitrary parent values
+      const parent_radius = 20;
+      const parent_cx = 175;
+      const parent_cy = 25;
+
+      // child circle calculations
+      const child_percent = child_value / parent_value;
+      const child_radius = parent_radius * child_percent;
+
+      return (
+        <svg viewBox="0 0 350 350">
+          <circle
+            cx={parent_cx.toString()}
+            cy={parent_cy.toString()}
+            r={parent_radius.toString()}
+            fill={color_scale(parent_name)}
+          />
+          <circle
+            cx={parent_cx.toString()}
+            cy={(parent_cy + parent_radius - child_radius).toString()}
+            r={child_radius.toString()}
+            fill={color_scale(child_name)}
+          />
+        </svg>
+      );
     };
 
-    const tooltip = () => (
-      <TooltipFactory
-        tooltip_items={[
-          {
-            id: "parent",
-            name: parent_name,
-            value: parent_value,
-            shape: "circle",
-            color: color_scale(parent_name),
-          },
-          {
-            id: "child",
-            name: child_name,
-            value: child_value,
-            shape: "circle",
-            color: color_scale(child_name),
-          },
-        ]}
-        tooltip_container_class="proportional-bubble-tooltip"
-        TooltipContentComponent={({ tooltip_item }) => (
-          <Fragment>
-            <MediaQuery minDeviceWidth={breakpoints.minMediumDevice}>
-              <td className="nivo-tooltip__label">{tooltip_item.name}</td>
-              <td className="nivo-tooltip__value">
-                {value_formatter(tooltip_item.value)}
-              </td>
-              <td className="nivo-tooltip__value">
-                {`(${formats.smart_percentage1_raw(
-                  tooltip_item.value / parent_value
-                )})`}
-              </td>
-            </MediaQuery>
-            <MediaQuery maxDeviceWidth={breakpoints.maxMediumDevice}>
-              <td>
-                <div className="nivo-tooltip__label ">{tooltip_item.name}</div>
-                <div className="nivo-tooltip__value">
-                  {value_formatter(tooltip_item.value)}
-                </div>
-                <div className="nivo-tooltip__value">
-                  {`(${formats.smart_percentage1_raw(
-                    tooltip_item.value / parent_value
-                  )})`}
-                </div>
-              </td>
-            </MediaQuery>
-          </Fragment>
-        )}
-      />
-    );
+    const Table = () => {
```

Comment:
For consistency with other graphs (pie chart at least), use the `TabularLegend` component instead.

Reply:
Done!